app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,12 @@
   <div>
     <%= msg %>
   </div>
+<% end %>
+<% if notice %>
+  <p class="alert alert-success"><%= notice %></p>
+<% end %>
+<% if alert %>
+  <p class="alert alert-danger"><%= alert %></p>
 <% end %>
```

Comment:
[OPTIONAL]

- I would recommend giving the `sign out` button as well if the user's login.
- Every time I do some event there are two notifications coming. Please fix it.

![Screenshot from 2022-03-02 22-00-38](https://user-images.githubusercontent.com/22434660/156405895-93719811-c480-4818-a548-c1aba066ee1c.png)